doc/learning-to-rank.md

```diff
@@ -58,7 +58,7 @@ This task needs to be run with access to Elasticsearch.  If you're
 using govuk-docker the full command will be:
 
 ```sh
-govuk-docker run -e ENABLE_LTR=1 search-api-lite bundle exec rake 'learn_to_rank:generate_training_dataset[judgements.csv]'
```

Comment:
Line 78 of this file also needs changing, and the bold warning at line 33 could be clarified too.

Reply:
I'm getting there! :-D